data/test-data/arrayConfigTest.yml

```diff
@@ -0,0 +1,10 @@
+site: North
+layoutName: Prod5
+modelVersion: Prod5
+default:
+    LST: '1'
```

Comment:
Took me a while to understand that the '1' is 'LST1' and not a numerical value (e.g., there should be 1 LST). Need to discuss names.